test/Microsoft.AspNetCore.WebSockets.Test/Program.cs

```diff
@@ -0,0 +1,17 @@
+#if NET461
```

Comment:
Wasn't able to quickly track down how we fixed this before. In `netcoreapp2.0` the Test SDK brings a dummy Program.Main, but in `net461` we need one to make the compiler happy (because of `<OutputType>exe</>`), thus the `#if`